scanner/src/test/kotlin/scanners/fossid/FossIdConfigTest.kt

```diff
@@ -52,6 +52,7 @@ class FossIdConfigTest : WordSpec({
                 "waitForResult" to "false",
                 "deltaScans" to "true",
                 "deltaScanLimit" to "42",
+                "timeout" to "300",
```

Comment:
Is the timeout now really in minutes, so this is 300 minutes?

Reply:
The timeout is now in minutes so 300 minutes -> 5 hours.
Or did I understood the question wrong ?

Reply:
Yes, I was just wondering about that huge timeout.